src/Qir/Execution/Tools/Driver/IQirSimulatorInitializer.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+
+namespace Microsoft.Quantum.Qir.Tools.Driver
+{
+    public interface IQirSimulatorInitializer
+    {
+        public string Generate();
+
+        public IList<string> Headers { get; }
```

Comment:
If this is only used to iterate elements from beginning to end (i.e. in foreach loop) consider making this IEnumerable<sting>. This will give more flexibility to implementers.

Reply:
Great recommendation! Changed it to `IEnumerable`.